.babelrc

```diff
@@ -5,12 +5,5 @@
         "node": "current"
       }
     }]
-  ],
-  "plugins": [
-    ["transform-runtime", {
-      "helpers": false,
-      "polyfill": false,
-      "regenerator": true
```

Comment:
I can remember having issues without it but it seems they solved them in the Jest core so thanks for the cleanup.

Reply:
hmmm never happened to me before and honestly I prefer to write test in ES5 except when the project is written in ES6

Reply:
No await/async was the deal breaker for me...

Reply:
I'm using async/await in https://github.com/luftywiranda13/generator-node-oss/blob/master/test.js

But I need no babel packages, I only set 'async-to-gen' as the transformer for jest

Reply:
Feel free to remove Babel dependency if it would still work for tests when!

Reply:
Okay, I'll continue hack on this when I got my laptop👍🏻 replying this with my phone now

Reply:
I realize it'll be easier to still write our test in ES2015+, so we don't have to change any logics/syntax and I see there's no downside with it